packages/data-sdk/src/utils/getCurrentModuleContext.ts

```diff
@@ -0,0 +1,8 @@
+export function getCurrentModuleContext(): string | null {
+  if (!(typeof window !== "undefined" && window.location)) {
+    return null;
+  }
+
+  const urlParams = new URLSearchParams(window.location.search);
+  return urlParams.get("module");
+}
```

Comment:
@alenjdev can the `module` property _change_ over the lifecycle of the app? Cause it asks for this in a _few_ places, but it _always_ re-parses the URL params to "figure it out". I was wondering if it was _possible_ for this to update this without reloading the page, and I dont _think_ thats a thing… but maybe i'm missing something.